caddy/caddymain/run.go

```diff
@@ -87,6 +89,13 @@ func Run() {
 		})
 	}
 
+	// initialize telemetry client
+	if enableTelemetry {
+		initTelemetry()
+	} else if disabledMetrics != "" {
+		mustLogFatalf("[ERROR] Cannot disable specific metrics because telemetry is disabled")
```

Comment:
Please explain how this is an unrecoverable state. If you cannot disable specific metrics because telemetry is disabled, who cares?

Reply:
> If you cannot disable specific metrics because telemetry is disabled, who cares?

It's a logical error. To not get an error is to imply that disabling metrics worked, which implies that telemetry is enabled, which is a contradiction.

Reply:
For these situations, rather than labeling as `[ERROR]`, it might be worth using something more appropriate (me, I tend to use `[INSANITY]`, and if it ever appears its easy to trap for and know that Things Have Gone Really Wrong).

Reply:
That's not appropriate @omz13